hackvote/app/controllers/projects_controller.rb

```diff
@@ -0,0 +1,41 @@
+class ProjectsController < ApplicationController
+	include ApplicationHelper
+
+	def create
+		@project = get_event.projects.build(project_params)
+		@project.project_id = @project.id
```

Comment:
The problem I was having here was that whenever I tried to do 
<%= button_to "vote", vote_path, params: { project_id: project.id }%>

and access the params through params[:tag], it would always be nil unless :tag was already existed in the model. And I can't use :id because I already used it to pass the event id through a hidden field.